src/content/docs/dropins/cart/cart-functions.mdx

```diff
@@ -298,118 +394,147 @@ type StoreConfigModel = {
+
+<CodeInclude code={cartModel} title="CartModel object" lang="ts" frame="none" />
+
+### Usage
+
+To reset the cart:
 
 ```js
 import { resetCart } from '@/cart/api/resetCart';
 
 resetCart();
 ```
 
-Here's the shape of the data returned from the `resetCart` function:
+## updateProductsFromCart
 
-```ts
-type Cart = {
-  cartId: null;
-  authenticated: false;
-};
+The `updateProductsFromCart` function updates the quantity of a product in the cart. The specified quantity replaces the current quantity. Setting the quantity to 0 removes an item from the cart. The function calls the [`updateCartItems`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/update-items/) mutation.
 
+```ts
+export const updateProductsFromCart = async (
+  items: UpdateProductsFromCart
+): Promise<CartModel | null>
 ```
 
-## updateProductsFromCart
+<OptionsTable
+  compact
+  options={[
+    ['Parameter', 'Type', 'Req?', 'Description'],
+    ['items','UpdateProductsFromCart','Yes','An input object that defines products to be updated.'],
+  ]}
+/>
+
+The `UpdateProductsFromCart` object has the following parameters:
+
+<OptionsTable
+  compact
+  options={[
+    ['Parameter', 'Type', 'Req?', 'Description'],
+    ['uid', 'string', 'Yes', 'The unique ID of an item in the cart.'],
+    ['quantity', 'number', 'Yes', 'The new quantity of the item.'],
+  ]}
+/>
+
+### Returns
+
+Returns a promise that resolves to a `CartModel` object or null. The `CartModel` object has the following shape:
 
-A function that updates products in cart. It takes the `uid` and `quantity` parameters.
+<CodeInclude code={cartModel} title="CartModel object" lang="ts" frame="none" />
+
+### Usage
+
+To changes the quantity of an item in the cart to 3.
 
 ```js
 import { updateProductsFromCart } from '@/cart/api/updateProductsFromCart';
 
 updateProductsFromCart([
   {
-    uid: string,
-    quantity: number,
+    uid: ''VA19-GO-NA,
+    quantity: 3,
   },
 ]);
 ```
 
-Here's the shape of the data returned from the `updateProductsFromCart` function:
+To remove the item from the cart.
 
-```ts
-type CartModel = {
-  id: string;
-  totalQuantity: number;
-  errors?: ItemError[];
-  items: Item[];
-  miniCartMaxItems: Item[];
-  total: {
-    includingTax: Price;
-    excludingTax: Price;
-  };
-  discount?: Price;
-  subtotal: {
-    excludingTax: Price;
-    includingTax: Price;
-    includingDiscountOnly: Price;
-  };
-  appliedTaxes: TotalPriceModifier[];
-  totalTax?: Price;
-  appliedDiscounts: TotalPriceModifier[];
-  shipping?: Price;
-  isVirtual?: boolean;
-  addresses: {
-    shipping?: {
-      countryCode: string;
-      zipCode?: string;
-      regionCode?: string;
-    }[];
-  };
-  isGuestCart?: boolean;
-}
+```js
+import { updateProductsFromCart } from '@/cart/api/updateProductsFromCart';
+
+updateProductsFromCart([
+  {
+    uid: 'VA19-GO-NA',
+    quantity: 2,
```

Comment:
**Issue (misleading):** This sets the quantity to 2. It would be more accurate to say this example decreases the quantity from 3 to 2 (if we're using the same context as the previous example). 

If we want this example to show how to remove an item completely from the cart, the quantity needs to be set to 0.